src/main/java/com/exasol/releaserobot/repository/maven/MavenPom.java

```diff
@@ -33,12 +47,43 @@ public String getVersion() {
     }
 
     /**
-     * Get deliverable name.
+     * Check if version presents.
+     *
+     * @return true if version presents
+     */
+    public boolean hasVersion() {
+        return this.version != null && !this.version.isEmpty();
+    }
+
+    /**
+     * Get properties map.
      * 
-     * @return deliverable name
+     * @return properties map
+     */
+    public Map<String, String> getProperties() {
+        return this.properties;
+    }
+
+    /**
+     * Check if properties present.
+     *
+     * @return true if properties present
+     */
+    public boolean hasProperties() {
+        return this.properties != null && !this.properties.isEmpty();
+    }
+
+    public List<MavenPlugin> getPlugins() {
+        return this.plugins;
+    }
+
+    /**
+     * Check if plugins present.
+     *
+     * @return true if plugins present
      */
-    public String getDeliverableName() {
-        return this.deliverableName;
+    public boolean hasPlugins() {
+        return this.plugins != null && !this.plugins.isEmpty();
```

Comment:
better avoid checking from null, and make sure the `plugins` collections is always there (even if it's empty). Probably you can do this in the builder